report/healthcheckreport_test.go

```diff
@@ -0,0 +1,39 @@
+package report
```

Comment:
Done. Check it out in this commit: https://github.com/frank0631/rhobot/commit/804f08f4b2cda51a93a46a0cfaff56ba618cc370